doc/userguide/rules/header-keywords.rst

```diff
@@ -293,6 +293,54 @@ Example of tos with a negated value:
 TCP keywords
 ------------
 
+tcp.flags
```

Comment:
would be good to add that this keyword can be used with `prefilter`, which can especially be helpful when having multiple similar rules.

e.g. `tcp.flags:FPU,CE; prefilter;`

Reply:
updated in next iteration.